spec/shadow/index.html

```diff
@@ -1260,9 +1260,15 @@
                 <li>Let <var>NAVIGATION-ORDER</var> be an empty list.</li>
                 <li>For each element <var>ELEMENT</var> in a <a>focus navigation scope</a> <var>SCOPE</var>,
                   <ol>
-                    <li>if <var>ELEMENT</var> is focusable, a <a>shadow host</a>, or a <a>slot element</a>, append <var>ELEMENT</var> to <var>NAVIGATION-ORDER</var>.</li>
+                    <li>If <var>ELEMENT</var> is a <a>shadow host</a>:
+                      <ol>
+                        <li>If its <a>tabindex</a> value is not negative and its <a>shadow root</a>'s delegatesFocus flag is set, append <var>ELEMENT</var> to <var>NAVIGATION-ORDER</var>.</li>
+                        <li>Otherwise, append <var>ELEMENT</var> to <var>NAVIGATION-ORDER</var>.</li>
```

Comment:
This is intentional.  Unless `delegatesFocus` flag is set, put the shadow host in the NAVIGATION-ORDER regardless of the tabindex value, to mark the point into which its shadow scope will be merged at the next step.